.envrc

```diff
@@ -0,0 +1,9 @@
+export PROJECTS_BUCKET=datalabs-data
+export PROJECT_NAME=grants_tagger
+
+export AWS_ACCOUNT_ID=160358319781
+export ECR_IMAGE=xxx
+export SAGEMAKER_ROLE=xxx
+
+export SLACK_HOOK=xxx
+export SLACK_USER=xxx
```

Comment:
did you mean to share this? better to populate the `.envrc.template` and let the user have their own `.envrc`. also note that if you populate the values git will detect a change if the file is git tracked.

Reply:
Oopsy. Fixed https://github.com/wellcometrust/grants_tagger/pull/147/commits/a435335ec02be91ca14630551f5e84063fc19325 and added a gitignore too